.github/workflows/integrate.yaml

```
@@ -65,16 +65,18 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - name: Setup operator environment
-      uses: charmed-kubernetes/actions-operator@main
+      uses: claudiubelu/actions-operator@18ebf92ae3043bd3dd15238e5d9b662d7ba08daf
```

Can we add this note (or similar)?
```suggestion
      # TODO: Unpin this when https://github.com/charmed-kubernetes/actions-operator/pull/46 is merged
      uses: claudiubelu/actions-operator@18ebf92ae3043bd3dd15238e5d9b662d7ba08daf
```